cpp/ql/lib/semmle/code/cpp/rangeanalysis/SimpleRangeAnalysis.qll

```diff
@@ -1594,7 +1603,7 @@ private module SimpleRangeAnalysisCached {
    * the lower bound of the expression after all the casts have been applied,
    * call `lowerBound` like this:
    *
-   *    `lowerBound(expr.getFullyConverted())`
+   *    lowerBound(expr.getFullyConverted())
```

Comment:
The indentation already turns on code highlighting, so the `'s just [end up](https://codeql.github.com/codeql-standard-libraries/cpp/semmle/code/cpp/rangeanalysis/new/SimpleRangeAnalysis.qll/predicate.SimpleRangeAnalysis$lowerBound.1.html) looking wonky in the rendered markdown.